charts/falco/values.yaml

```diff
@@ -708,13 +715,13 @@ falco:
     ca_bundle: ""
     # -- Path to a folder that will be used as the CA certificate store. CA certificate need to be
     # stored as indivitual PEM files in this directory.
-    ca_path: "/etc/ssl/certs"
+    ca_path: "/etc/falco/certs/"
     # -- Tell Falco to use mTLS
     mtls: false
     # -- Path to the client cert.
-    client_cert: "/etc/ssl/certs/client.crt"
+    client_cert: "/etc/falco/certs/client/client.crt"
     # -- Path to the client key.
-    client_key: "/etc/ssl/certs/client.key"
+    client_key: "/etc/falco/certs/client/client.key"
     # -- Whether to echo server answers to stdout
```

Comment:
Although this is legit, we will risk getting these values overwritten when maintainers update the config (N.B.: these defaults are usually copied/pasted from [falco.yaml](https://github.com/falcosecurity/falco/blob/master/falco.yaml)).

 So I'd prefer to keep the `/etc/ssl/certs` folder unless there's any compelling reason not to do so :thinking:

Reply:
Reason I changed the path is because if we use /etc/ssl/certs we are overwriting the base image CAs and other certificates, risking getting certificate errors if the container need to trust other servers or other certificate related issues.

This being said, we could mount a folder inside like /etc/ssl/certs/ falco so there's some consistency :)

Reply:
Good point. I believe `/etc/falco/certs` works. Just waiting for @alacuku for a second opinion.

Thank you!